dist-git.spec

```diff
@@ -258,12 +258,13 @@ fi
 
 %dir              %{_datadir}/dist-git
 %attr (775, -, -) %{_datadir}/dist-git/*
+%{_bindir}/*
```

Comment:
Unrelated but since you are changing the spec, can you please make the `%{_bindir}/*` more specific?

Reply:
Updated.